content/en/logs/guide/collect-google-cloud-logs.md

```diff
@@ -0,0 +1,155 @@
+5. Under **Retry policy**, select `Retry after exponential backoff delay`.
+6. Hit `Create` at the bottom.
+
+The Pub/Sub is ready to receive logs from Google Cloud Logging and forward them to Datadog.
+
+#### Export logs from Google Cloud
+
+1. Go to [the Logs Explorer page][6] and filter the logs that need to be exported.
+2. From **Log Router** tab, select **Create Sink**.
+3. Provide a name for the sink.
+4. Choose _Cloud Pub/Sub_ as the destination and select the pub/sub that was created for that purpose. **Note**: The pub/sub can be located in a different project.
+
+    {{< img src="integrations/google_cloud_pubsub/creating_sink2.png" alt="Export Google Cloud Pub/Sub Logs to Pub Sub" >}}
+
+5. Click **Create Sink** and wait for the confirmation message to show up.
+
+**Note**: It is possible to create several exports from Google Cloud Logging to the same Pub/Sub with different sinks.
+
+**Warning**: Pub/subs are subject to [Google Cloud quotas and limitations][7]. If the number of logs you have is higher than those limitations, Datadog recommends you split your logs over several topics. See [the Monitor the Log Forwarding section](#monitor-the-log-forwarding) for information on how to setup a monitor to be automatically notified if you get close to those limits.
+
+### Datadog Dataflow template
+
+The Datadog Dataflow template also integrates with a Google Cloud Pub/Sub, but uses a **Pull** delivery type instead of **Push**.
+
+1. Configure a Cloud Pub/Sub subscription with the **Pull** delivery type.
+
+**Note**: The default `Compute Engine` service account needs access to read from this subscription. Ensure the following roles are included:
+   - Pub/Sub Viewer
+   - Pub/Sub Subscriber
+
+2. Click the checkbox to **Enable dead lettering** and select or create a topic to server as a dead-letter queue (DLQ) in the case of failures.
+
+**Note**: The default `Compute Engine` service account needs access to write to this topic. Ensure the following roles are included:
+   - Pub/Sub Publisher
+
+3. Create a new secret in Secret Manager with your valid Datadog API Key.
+
+**Note**: The default `Compute Engine` service account needs access to write to this topic. Ensure the following roles are included:
+   - Secret Manager 
+   - Secret Accessor
+
+4. Create a [dataflow job][11]. Datadog's dataflow template provides both compression and batching of events before forwarding to Datadog.
+
+You should set Dataflow parameters based on your scale. If you're not sure where to begin, use the example parameter values included below, and make adjustments based on load tests.
+
+**Note**: Use the **site parameter** for your Datadog site listed in [Getting Started with Datadog Sites][12] as the value for `<SITE_PARAMETER>`.
+
+{{< code-block lang="yaml" filename="" disable_copy="true" collapsible="true" >}}
+
+<strong>Generic Dataflow Parameters</strong>
```

Comment:
I think we don't want to provide guidance on these default Dataflow parameters from our main document page.

Reply:
This is mostly shared with customers privately so they can get started but it wouldn't do well as a published set of params. We are working with Google instead to publish a document such as this in their architecture center -- 

https://cloud.google.com/architecture/deploying-production-ready-log-exports-to-splunk-using-dataflow#set_maximum_pipeline_size

Reply:
Thanks for confirming that, removed this section in [0bba4c6](https://github.com/DataDog/documentation/pull/19467/commits/0bba4c6a103c50b58d06acefa33ded179241fadb)